gui/src/main/java/io/bisq/gui/main/settings/network/NetworkSettingsView.java

```diff
@@ -109,15 +110,23 @@
     private ChangeListener<Filter> filterPropertyListener;
 
     @Inject
-    public NetworkSettingsView(WalletsSetup walletsSetup, P2PService p2PService, Preferences preferences, BitcoinNodes bitcoinNodes,
-                               FilterManager filterManager, BisqEnvironment bisqEnvironment, Clock clock, BSFormatter formatter) {
+    public NetworkSettingsView(WalletsSetup walletsSetup,
+                               P2PService p2PService,
+                               Preferences preferences,
+                               BitcoinNodes bitcoinNodes,
+                               FilterManager filterManager,
+                               BisqEnvironment bisqEnvironment,
+                               TorNetworkSettingsWindow torNetworkSettingsWindow,
+                               Clock clock,
+                               BSFormatter formatter) {
```

Comment:
I don't think we want to move to one-parameter per line parameter stacking like this. Creates a lot of vertical noise, and constructors with a zillion parameters are usually a signal of something needing to be refactored anyway. I'd prefer that we keep parameters aligned with the opening paren, but not stack one per line. Just go to the 120 margin, and create a new line as necessary.

Reply:
Re typos: 
Fixed in next commit.

Re Popup: 
The popup is used when Tor cannot start up after 4 minutes at splash screens. So that is a reason why a dedicated Tor view will not work here. It is also considered as "exceptional" screen which should not be needed for normal use case. 

Re why it changes anything:
I agree it is not satisfying not knowing more details. Will be a task for our new P2P network dev. There are some consensus files which might get outdated. I just have seen it yesterday with a seed node which kept failing after repeated restarts and after deleting the Tor dir it was fine.

Re ctor params: 
I personally prefer it for readability to have one per line if its more then 3 or 4 params...
But not super strong opinion here. we can discuss... I prefer to keep it atm as I usually use it that way in most of the newer code parts.

Re deleteDirectory:
Do you see any risks in that method?

Reply:
Makes sense, re popup. I didn't realize that before. Thanks.

On `deleteDirectory` I didn't analyze it line by line, but gave it a good look and then tested it end-to-end on my machine. This is the only place it's used in the codebase, so I'd say the risk is contained for now. Would be good to have tests around it for sure, but I'm ok with it as-is now.